x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -688,13 +718,24 @@ export const ModelsList: FC<Props> = ({
     <>
       <SavedObjectsWarning onCloseFlyout={fetchModelsData} forceRefresh={isLoading} />
       <EuiFlexGroup justifyContent="spaceBetween">
-        {modelsStats && (
-          <>
-            <EuiFlexItem grow={false}>
-              <StatsBar stats={modelsStats} dataTestSub={'mlInferenceModelsStatsBar'} />
-            </EuiFlexItem>
-          </>
-        )}
+        {modelsStats ? (
+          <EuiFlexItem grow={false}>
+            <StatsBar stats={modelsStats} dataTestSub={'mlInferenceModelsStatsBar'} />
+          </EuiFlexItem>
+        ) : null}
+        <EuiFlexItem grow={false}>
+          <EuiButton
+            fill
+            iconType={'plusInCircle'}
+            color={'primary'}
+            onClick={setIsAddModelFlyoutVisible.bind(null, true)}
+          >
+            <FormattedMessage
+              id="xpack.ml.trainedModels.modelsList.addModelButtonLabel"
+              defaultMessage="Add trained model"
```

Comment:
For users with read-only feature privileges for ML, we need to hide some or all of this functionality. The first tab will need to be hidden as users won't be able to download the models. 

Showing the info for using Eland is probably ok, as it's simply showing info which is available in the docs.

Reply:
Updated in https://github.com/elastic/kibana/pull/171024/commits/004d2761836e6b2464b2ee80acf3cdf6acc2251d

Reply:
For a user with read-only feature privileges for ML I am seeing a blank 'Third-party' tab:

<img width="367" alt="image" src="https://github.com/elastic/kibana/assets/7405507/4841c34e-9224-4548-940d-f547c76f0f60">

Reply:
Fixed in https://github.com/elastic/kibana/pull/171024/commits/ebde32935b8ab8499c0531409e37a96c207e624a